### PR TITLE
Upgraded to play 2.7.3 and slick 3.3.2

### PR DESCRIPTION
**Pull request by KacperFKorban**

### CODE REVIEW

File: project/Settings.scala [hunk 1]
```diff
@@ -19,6 +19,7 @@ object Settings {
     crossScalaVersions := Seq(scala_2_11, scala_2_12),
     releaseCrossBuild := true,
 
+    fork in Test := true,
```
Comment (bartosz822) [inline project/Settings.scala]: why this change?

File: project/plugins.sbt [hunk 2]
```diff
@@ -26,3 +26,11 @@ addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.4")
 
 /*---------------*/
+/* Play */
+
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.3")
```
Comment (bartosz822) [inline project/plugins.sbt]: is this needed?

Comment (KacperFKorban) [reply project/plugins.sbt]: It's not. Removed.

File: project/Settings.scala (hunk 1 above)
Comment (KacperFKorban) [reply project/Settings.scala]: This line fixes tests execution with scala 2.11. It's a Scala issue. Scala links wrong version of scala-parser-combinators.

File: project/Dependencies.scala [hunk 3]
```diff
@@ -2,26 +2,27 @@ import sbt._
...
-    "org.scalatest" %% "scalatest" % "3.0.1" % "test",
-    "com.h2database" % "h2" % "1.4.187" % "test",
-    "ch.qos.logback" % "logback-classic" % "1.1.3" % "test"
+    "org.scalatest" %% "scalatest" % "3.0.8" % "test",
+    "com.h2database" % "h2" % "1.4.199" % "test",
+    "ch.qos.logback" % "logback-classic" % "1.2.3" % "test"
   )
 
-  def core(scalaVersion: String) = mainCore(scalaVersion) ++ testCore
+  def core(scalaVersion: String): Seq[ModuleID] = mainCore(scalaVersion) ++ testCore
```
[the remark this answers is not in the document]
Comment (KacperFKorban) [reply project/Dependencies.scala]: Done.

File: project/Dependencies.scala [hunk 4]
```diff
@@ -2,26 +2,27 @@ import sbt._
 
 object Dependencies {
 
-  def mainCore(scalaVersion: String) = Seq(
-    "com.typesafe.slick" %% "slick" % "3.2.3",
+  def mainCore(scalaVersion: String): Seq[ModuleID] = Seq(
```
[the remark this answers is not in the document]
Comment (KacperFKorban) [reply project/Dependencies.scala]: Done.

File: project/Settings.scala (hunk 1 above)
Comment (bartosz822) [reply project/Settings.scala]: ok, makes sense
